tests/conftest.py

```diff
@@ -32,3 +33,21 @@ def mock_data_client():
     with open(os.path.join(FIXTURES_DIR, 'test_supplier_frameworks_response.json')) as supplier_frameworks_response:
         mock_data_client.find_framework_suppliers.return_value = json.loads(supplier_frameworks_response.read())
     return mock_data_client
+
+
+class MockResponse:
```

Comment:
I like this simple class, but is there a reason we're not using `requests-mock` (like we do in the API client?)

Reply:
None! I just didn't know we used that. I'll change over to it.